python/pyarrow/_rados.pyx

```diff
@@ -22,6 +22,17 @@ from pyarrow.lib cimport *
 from pyarrow.lib import frombytes, tobytes
 
 cdef class RadosParquetFileFormat(FileFormat):
+    """
+    A ParquetFileFormat implementation that offloads the fragment
+    scan operations to the Ceph OSDs
+
+    Parameters
+    ---------
+    ceph_config_path: The path to the ceph config file.
```

Comment:
Really nit-pick, but if we are using "Ceph" somewhere and "ceph" somewhere, that does not look good. Let's use "Ceph" everywhere.

Reply:
Done.